src/CMakeLists.txt

```diff
@@ -323,7 +323,10 @@ elseif ( WIN32 )
   set_target_properties ( libfluidsynth
     PROPERTIES
       PUBLIC_HEADER "${public_HEADERS}"
+      PREFIX "lib"
       OUTPUT_NAME "fluidsynth-${LIB_VERSION_CURRENT}"
+      IMPORT_PREFIX "lib"
+      ARCHIVE_OUTPUT_NAME "fluidsynth-${LIB_VERSION_CURRENT}"
```

Comment:
This change causes static or import libraries on Windows to always be versioned.
So it's currently impossible to link to fluidsynth as `-lfluidsynth`.

See [the contents of the msys2 mingw64 fluidsynth package](https://packages.msys2.org/packages/mingw-w64-x86_64-fluidsynth): no `libfluidsynth.a` or `libfuildsynth.dll.a` is present.